application/cloud_api.py

```diff
@@ -63,42 +115,239 @@ def uploadFiles(self, types, paths, callback):
+        r = requests.post(self.server_addr + 'highlightVideo', data = payload)
+        print "Status Code: {}".format(r.status_code)
+        print "Response Text: {}".format(r.text)
+
+    def makeReport(self, identifier):
+        print "makeReport called with identifier = {}".format(identifier)
+
+        payload = {
+            'identifier': identifier,
+        }
+
+        r = requests.post(self.server_addr + 'makeReport', data = payload)
+        print "Status Code: {}".format(r.status_code)
+        print "Response Text: {}".format(r.text)
+
+    def retrieveResults(self, identifier):
+        print "retrieveResults called with identifier = {}".format(identifier)
+
+        payload = {
+            'identifier': identifier,
+        }
+        local_filename = 'highlight_video.mp4'
+        r = requests.get(self.server_addr + 'retrieveResults', data = payload, stream=True)
+        with open(local_filename, 'wb') as f:
+            print('Dumping "{0}"...'.format(local_filename))
+            for chunk in r.iter_content(chunk_size=2048):
+                if chunk:
+                    f.write(chunk)
+        print file_path
+        print "Status Code: {}".format(r.status_code)
+
+    def roadUserCounts(self, identifier):
+        print "roadUserCounts called with identifier = {}".format(identifier)
+
+        payload = {
+            'identifier': identifier,
+        }
+
+        r = requests.post(self.server_addr + 'roadUserCounts', data = payload)
+        print "Status Code: {}".format(r.status_code)
+        print "Response Text: {}".format(r.text)
+
+    def speedCDF(self, identifier, speed_limit = None, vehicle_only = None):
+        print "speedCDF called with identifier = {}, speed_limit = {} and vehicle_only = {}"\
+                .format(identifier, speed_limit, vehicle_only)
+
+        payload = {
+            'identifier': identifier,
+            'speed_limit': speed_limit,
+            'vehicle_only': vehicle_only
+        }
+
+        r = requests.post(self.server_addr + 'speedCDF', data = payload)
+        print "Status Code: {}".format(r.status_code)
+        print "Response Text: {}".format(r.text)
 
-    def generateDefaultConfig(self, callback):
-        raise NotImplementedError("generateDefaultConfig not yet implemented")
 
 #FOR TESTING PURPOSES ONLY
 if __name__ == '__main__':
-    remote = CloudWizard('10.7.90.25')
-    local = CloudWizard('localhost')
-    #server = TrafficCloud_Server("/project/path/goes/ahere","localhost")
-    print remote.server_ip
-    print remote.readIds()
-    print local.server_ip
-    print local.readIds()
\ No newline at end of file
+    print "Syntax looks fine!"
+
+    ###########################################################################
+    # Setup CloudWizard
+    ###########################################################################
+    #remote = CloudWizard('10.7.88.26')
+    #remote = CloudWizard('10.26.89.15')
+    remote = CloudWizard('10.7.24.11')
```

Comment:
I would probably not commit with a personal ip address in the file. Maybe make this read in from a file to hide it. Or use a environment variable. When we have AWS up, then this could direct there.